src/@providers/gcp/services/database-version.ts

```diff
@@ -1,4 +1,4 @@
-import { Auth, google } from 'googleapis';
+import { Auth, google } from 'npm:googleapis';
```

Comment:
Similar question as before - we should have a version of this pinned in the import map, any reason to be doing this instead?